lib/services/runtime/handlers/cardV2/cardV2.alexa.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Alexa cardV2 needs to be used in favor of general cardV2 because
+ * it uses different command handler
+ * it uses different no match handler
+ */
+import { VoiceflowConstants, VoiceflowNode } from '@voiceflow/voiceflow-types';
+
+import { HandlerFactory } from '@/runtime';
+
+import { CommandAlexaHandler } from '../command/command.alexa';
+import { NoMatchAlexaHandler } from '../noMatch/noMatch.alexa';
+import { CardV2Handler, handlerUtils } from './cardV2';
+
+const utils = {
+  ...handlerUtils,
+  commandHandler: CommandAlexaHandler(),
+  noMatchHandler: NoMatchAlexaHandler(),
+};
+
+export const CardV2AlexaHandler: HandlerFactory<VoiceflowNode.CardV2.Node, typeof handlerUtils> = (utils) => {
+  const cardV2AlexaHandler = CardV2Handler(utils);
+  return {
+    ...cardV2AlexaHandler,
+    canHandle: (node, ...args) =>
+      cardV2AlexaHandler.canHandle(node, ...args) && node.platform === VoiceflowConstants.PlatformType.ALEXA,
```

Comment:
We should check the platform first to exit early and avoid any additional checks if it's not the matching platform.

We could also set the `handle` directly to make it a bit more explicit about what this is returning.

```suggestion
  const {handle, canHandle} = CardV2Handler(utils);
  return {
    handle,
    canHandle: (node, ...args) =>
     node.platform === VoiceflowConstants.PlatformType.ALEXA && canHandle(node, ...args),
```

Same idea for the other handles with this same pattern.

Reply:
good call, explicit is always better than implicit!
will make those changes :)